presto-docs/src/main/sphinx/troubleshoot.rst

```diff
@@ -0,0 +1,14 @@
+***************
+Troubleshooting
+***************
+
+.. toctree::
+    :maxdepth: 1
+
+    troubleshoot/deploy
+
+This section of the Presto documentation collects known problems when working 
```

Comment:
Would it make sense to move this above the links? Maybe reword to something like:
"The links below are common problems when working with Presto..."